includes/functions.php

```diff
@@ -227,6 +227,14 @@ function tsml_custom_addresses($custom_overrides)
 	$tsml_google_overrides = array_merge($tsml_google_overrides, $custom_overrides);
 }
 
+//fuction:	define custom type descriptions
+//used:		theme's functions.php
+function tsml_custom_descriptions($descriptions)
```

Comment:
thing you might need to merge the branch with main

Reply:
i like to use the [github desktop app](https://desktop.github.com/) which makes this super easy. but from the command line i think you can just do `git merge main` and then push the change with `git push origin` and this change should go away

Reply:
???

Reply:
Update branding reset to Refresh.

Reply:
is the ??? because the instructions aren't clear, or the issue isn't clear? 

the issue is that this is showing up as code you're adding to the branch, but it's already in `main`

i think if you update git it will sync the changes properly

Reply:
Sorry, I'm multi-tasking on a zoom with my siblings. I've merged main into my branch and committed some changes into the branch. I'll do another one tomorrow to get the rest of your review items in.